sale_cancel_reason/wizard/__init__.py

```diff
@@ -3,6 +3,7 @@
 #
 #    Author: Guewen Baconnier
 #    Copyright 2013 Camptocamp SA
+#    Copyright 2016 Serpent Consulting Services Pvt. Ltd.
```

Comment:
Don't add your copyright on non touched files (and more if they are `__init__.py` files.